src/Sylius/Component/Core/Provider/LastYearSalesSummaryProviderInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Provider;
+
+use Sylius\Component\Core\Dashboard\SalesSummaryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+
+interface LastYearSalesSummaryProviderInterface
```

Comment:
Is it really `LastYear`? It feels like I could pass any start and end dates here. I think we can go with `SalesSummaryProviderInterface`.